go/cli/mcap/cmd/cat.go

```diff
@@ -237,5 +248,7 @@ func init() {
 	catCmd.PersistentFlags().Int64VarP(&catStart, "start-secs", "", 0, "start time")
 	catCmd.PersistentFlags().Int64VarP(&catEnd, "end-secs", "", math.MaxInt64, "end time")
 	catCmd.PersistentFlags().StringVarP(&catTopics, "topics", "", "", "comma-separated list of topics")
-	catCmd.PersistentFlags().BoolVarP(&catFormatJSON, "json", "", false, "print messages as JSON")
+	catCmd.PersistentFlags().BoolVarP(&catFormatJSON, "json", "", false,
+		`print messages as JSON. Supported schema encodings: ros1msg, protobuf, and jsonschema. 
```

Comment:
```suggestion
		`print messages as JSON. Supported message encodings: ros1msg, protobuf, and JSON.`)
```

Reply:
ros1msg isn't a message encoding, it's a schema encoding. Also would rather see `json` than `JSON` since that's what the actual value of the encoding string is.